renovate.json

```diff
@@ -13,6 +14,16 @@
     {
       "depTypeList": ["peerDependencies"],
       "enabled": false
+    },
+    {
+      "baseBranchList": ["master"],
+      "updateTypes": ["major"],
+      "enabled": false
+    },
+    {
+      "baseBranchList": ["next"],
+      "updateTypes": ["digest", "pin", "patch", "minor"],
+      "enabled": false
```

Comment:
hm, sometimes major released can be merged in master, because they don't have breaking changes